network/hostnetwork/rpc/factory.go

```diff
@@ -32,7 +32,7 @@ func NewRPCFactory(methods map[string]RemoteProcedure) Factory {
 	}
 }
 
-// Create creates and registers new remote procedure.
+// create creates and registers new remote procedure.
```

Comment:
https://blog.golang.org/godoc-documenting-go-code

«The convention is simple: to document a type, variable, constant, function, or even a package, write a regular comment directly preceding its declaration»